### DISCUSSION TITLE
¿Como crear un brand brick?

### DISCUSSION THREAD

**Discussion by JMRodriguez-work** [mercadopago/sdk-react#103] [Q&A] [answered]
## Alguien me puede guiar un poco en cómo utilizar el Brand?
Vi en la documentación que lo importan de manera nombrada { Brand } pero no me aparece, en otras partes vi que se utiliza un
```
bricksBuilder.create(
    "brand",
    "brandBrick_container"
  );
};
```
pero no estoy seguro en qué parte de mi código debo utilizarlo.

**Comment by icaldana** [comment, marked as answer]
Buenas, como está?
Leíste por casualidad los pasos [inicio común](https://www.mercadopago.com.ar/developers/es/docs/checkout-bricks/common-initialization)? Existe tambien la [live demo](https://www.mercadopago.com.ar/developers/es/live-demo/brand-brick) para hacer pruebas y incluso puedes copiar el código desde allí.

Muchas gracias!
